clients/php/predis.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "predis",
+    "description": "A flexible and feature-complete Redis client for PHP.",
+    "twitter": [
+        "JoL1hAHN"
+    ], 
+    "repo":"https://github.com/predis/predis",
+    "installation": "composer require predis/predis",
+    "language":"php",
+    "package_size": "2.8M",
+    "license": "MIT",
+    "read_from_replica": true,
+    "smart_backoff_to_prevent_connection_storm": true,
+    "pubsub_state_restoration": false,
+    "cluster_scan": false,
+    "latency_based_read_from_replica": false,
+    "AZ_based_read_from_replica": false,
+    "client_side_caching": true,
```

Predis doesn't really support client-side caching, only when [Relay](http://github.com/cachewerk/relay) is used.

```suggestion
    "client_side_caching": false,
```